sys/linux/dev_img_rogue.txt

```diff
@@ -0,0 +1,1925 @@
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_DEVMEMINTUNPININVALIDATE {
+	hMapping	IMG_HANDLE
+	hPMR		IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_DEVMEMINTUNPININVALIDATE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_DEVMEMINTUNRESERVERANGE {
+	hReservation	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_DEVMEMINTUNRESERVERANGE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_DEVMEMINVALIDATEFBSCTABLE {
+	hDevmemCtx	IMG_HANDLE
+	ui64FBSCEntries	int64
+} [packed]
+
+PVRSRV_BRIDGE_OUT_DEVMEMINVALIDATEFBSCTABLE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_DEVMEMISVDEVADDRVALID {
+	hDevmemCtx	IMG_HANDLE
+	sAddress	IMG_DEV_VIRTADDR
+} [packed]
+
+PVRSRV_BRIDGE_OUT_DEVMEMISVDEVADDRVALID {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_GETMAXDEVMEMSIZE {
+	ui32EmptyStructPlaceholder	const[0, int32]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_GETMAXDEVMEMSIZE {
+	uiLMASize	int64
+	uiUMASize	int64
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_HEAPCFGHEAPCONFIGCOUNT {
+	ui32EmptyStructPlaceholder	const[0, int32]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_HEAPCFGHEAPCONFIGCOUNT {
+	ui32NumHeapConfigs	int32
+	eError			int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_HEAPCFGHEAPCONFIGNAME {
+	ui32HeapConfigIndex	int32
+	ui32HeapConfigNameBufSz	len[puiHeapConfigName, int32]
+	puiHeapConfigName	ptr[in, array[int8]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_HEAPCFGHEAPCONFIGNAME {
+	puiHeapConfigName	ptr[out, array[int8]]
+	eError			int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_HEAPCFGHEAPCOUNT {
+	ui32HeapConfigIndex	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_HEAPCFGHEAPCOUNT {
+	ui32NumHeaps	int32
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_HEAPCFGHEAPDETAILS {
+	ui32HeapConfigIndex	int32
+	ui32HeapIndex		int32
+	ui32HeapNameBufSz	len[puiHeapNameOut, int32]
+	puiHeapNameOut		ptr[in, array[int8]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_HEAPCFGHEAPDETAILS {
+	puiHeapNameOut			ptr[out, array[int8]]
+	sDevVAddrBase			IMG_DEV_VIRTADDR
+	uiHeapLength			int64
+	uiReservedRegionLength		int64
+	ui32Log2DataPageSizeOut		int32
+	ui32Log2ImportAlignmentOut	int32
+	eError				int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_PHYSMEMNEWRAMBACKEDLOCKEDPMR {
+	uiSize			int64
+	uiChunkSize		int64
+	ui32NumPhysChunks	int32
+	ui32NumVirtChunks	len[pui32MappingTable, int32]
+	pui32MappingTable	ptr[in, array[int32]]
+	ui32Log2PageSize	int32
+	uiFlags			flags[PVRSRV_MEMALLOCFLAGS_T, int32]
+	ui32AnnotationLength	len[puiAnnotation, int32]
+	puiAnnotation		ptr[in, array[int8]]
+	ui32PID			int32
+	ui32PDumpFlags		int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_PHYSMEMNEWRAMBACKEDLOCKEDPMR {
+	hPMRPtr	IMG_HANDLE
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_PHYSMEMNEWRAMBACKEDPMR {
+	uiSize			int64
+	uiChunkSize		int64
+	ui32NumPhysChunks	len[pui32MappingTable, int32]
+	ui32NumVirtChunks	int32
+	pui32MappingTable	ptr[in, array[int32]]
+	ui32Log2PageSize	int32
+	uiFlags			flags[PVRSRV_MEMALLOCFLAGS_T, int32]
+	ui32AnnotationLength	len[puiAnnotation, int32]
+	puiAnnotation		ptr[in, array[int8]]
+	ui32PID			int32
+	ui32PDumpFlags		int32
```

Comment:
It looks like the possible values for ui32PDumpFlags are PDUMP_NONE, PDUMP_BLKDATA, PDUMP_CONT, PDUMP_PERSIST. There seem to be some more inside drivers/gpu/drm/img-rogue/1.10/pdump.h as PDUMP_FLAGS_PERSISTENT, PDUMP_FLAGS_DEBUG, PDUMP_FLAGS_NOHW, PDUMP_FLAGS_FORCESPLIT.

Reply:
Fixed.